src/runtime_src/core/include/xrt.h

```diff
@@ -200,11 +223,11 @@ struct xclDeviceUsage {
     size_t h2c[8];
     size_t c2h[8];
     size_t ddrMemUsed[8];
-    unsigned ddrBOAllocated[8];
-    unsigned totalContexts;
+    unsigned int ddrBOAllocated[8];
```

Comment:
Instead of "int" should this value be a long for on linux an int is 64bits and windows it is 32bits.
There seems to be a few of these items, so I will not repeat this comment in this file :).

Reply:
I don't have the liberty to change the size of this struct on Linux.  'unsigned' is a misnomer for 'unsigned int' so I essentially didn't change anything , which is what I wanted.